src/deadline/unreal_submitter/unreal_open_job/unreal_open_job_step.py

```diff
@@ -324,7 +438,16 @@ def mrq_job(self, value: unreal.MoviePipelineExecutorJob):
 
     def _get_chunk_ids_count(self) -> int:
         """
-        Get parameters
+        Get number of shot chunks
+        as count of enabled shots in level sequence divided by chuck size parameter value.
+        If no chunk size parameter value is 0, return 1
+
+        Example:
+            - LevelSequence shots: [sh1 - enabled, sh2 - disabled, sh3 - enabled, sh4 - enabled]
+            - ChunkSize: 2
+            - ChunkIds count: 2 (sh1, sh3; sh4)
+
+        :return: list of ChunkIds, e.g. [0, 1, 2, 3]
```

Comment:
Looks like this returns the count rather than list

Reply:
oops, yes, will fix it